.github/CONTRIBUTING.md

```diff
@@ -86,6 +86,17 @@ Also mind that CoffeeScript is production dependency (not dev dependency),
 because it's needed not only for compiling Dredd package before uploading
 to npm, but also for running user-provided hooks written in CoffeeScript.
 
+### Compiled vs pure JavaScript
+
+Dredd uses [Drafter][] for parsing [API Blueprint][] documents. Drafter is written in C++11 and needs to be compiled during installation. Since that can be problematic for some environments and leads to a lot of troubleshooting, there's also pure JavaScript version of the parser, [drafter.js][]. While drafter.js is fully equivalent, it can have slower performance. That's why there's [drafter-npm][] package, which first tries to compile the C++11 version of the parser and if it's unsuccessful, uses the JavaScript equivalent.
```

Comment:
`Because this can cause a lot of troubles in some environments, there's also...`
`Drafter.js is fully equivalent, but it can have slower performance.`
`Therefore there's...`
`which tries to compile the C++11 version of the parser and uses the JavaScript equivalent in case of failure`